receiver/simpleprometheusreceiver/config.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package simpleprometheusreceiver
+
+import (
+	"time"
+
+	"github.com/open-telemetry/opentelemetry-collector/config/configmodels"
+)
+
+// Config defines configuration for simple prometheus receiver.
+type Config struct {
+	configmodels.ReceiverSettings `mapstructure:",squash"`
+	httpOptionsConfig             `mapstructure:",squash"`
+	// CollectionInterval is the interval at which metrics should be collected
+	CollectionInterval time.Duration `mapstructure:"collection_interval"`
+	// MetricsPath the path to the metrics endpoint.
+	MetricsPath string `mapstructure:"metrics_path"`
+	// Whether or not to use pod service account to authenticate.
+	UseServiceAccount bool   `mapstructure:"use_service_account"`
+}
+
+// TODO: Move to a common package for use by other receivers and also pull
+// in other utilities from
+// https://github.com/signalfx/signalfx-agent/blob/master/pkg/core/common/httpclient/http.go.
+type httpOptionsConfig struct {
+	// Whether or not to use HTTPS instead of HTTP.
+	UseHTTPS          bool   `mapstructure:"use_https"`
+	// Path to the CA cert that has signed the TLS cert.
+	CAFilePath        string `mapstructure:"ca_file_path"`
+	// Path to the client TLS cert to use for TLS required connections.
+	CertFilePath      string `mapstructure:"cert_file_path"`
+	// Path to the client TLS key to use for TLS required connections.
+	KeyFilePath       string `mapstructure:"key_file_path"`
+	// If `use_https` is true and this option is also true, the exporter's TLS
+	// cert will not be verified.
+	SkipVerify        bool   `mapstructure:"skip_verify"`
```

Comment:
```suggestion
	InsecureSkipVerify        bool   `mapstructure:"insecure_skip_verify"`
```
It is not clear what "skip_verify" refers to.

Reply:
Updated.